### PR TITLE
Update CODEOWNERS

### PR DESCRIPTION
**Pull request by tdykstra**
Assign character encoding docs to tdykstra

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -143,6 +143,9 @@
 /docs/standard/asynchronous-programming-patterns/ @BillWagner
 # Attributes
 /docs/standard/attributes/ @tdykstra
+# Character encoding
+/docs/standard/base-types/character-encoding / @tdykstra
```
Comment (gewarren) [inline .github/CODEOWNERS:147]:
```suggestion
/docs/standard/base-types/character-encoding/ @tdykstra
```

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -143,6 +143,9 @@
...
+/docs/standard/base-types/character-encoding-introduction / @tdykstra
```
Comment (gewarren) [inline .github/CODEOWNERS:148]:
```suggestion
/docs/standard/base-types/character-encoding-introduction/ @tdykstra
```

**Review by gewarren — APPROVED** [review]
Approved with changes.

**Comment by tdykstra** [comment]
Thanks for the review and corrections, @gewarren!

**Pull request merged**